source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+white-lists the intended traffic for that port.
+
+### User interface
+
+The user is meant to provision security group(s) and rule(s) through ansible,
+or using a REST client.
+
+This greatly reduces the oVirt-engine impact, since there's no need to 
+implement the REST client side of the API, nor implement these changes in the
+GUI.
+
+Throughout the remainder of the document, ansible will be assumed as the client, given its descriptive yaml syntax. The [os_security_group](https://docs.ansible.com/ansible/2.5/modules/os_security_group_module.html)
+and [os_security_group_rules](https://docs.ansible.com/ansible/2.5/modules/os_security_group_rule_module.html)
+will be leveraged for this.
+
+## Activating the feature
+
+The feature will be always active, and it is a matter of identifying the
+ports that will be subject to the security group filters.
+
+As previously mentioned, the **port-security** attribute will be used to
+know if a port will be added to the **deny-all** port group, having all
+ingress IP traffic dropped as a consequence.
+
+The **port-security** attribute is set at port creation time, and its value will
+be inherited from the corresponding *network* entity when not defined at port
+level. Afterwards, it can be updated through the API - at port *or* network
+levels.
+
+The default behavior for the network's **port-security** attribute would be
+defined in the configuration file, through the ***port-security-enabled***,
+property, located in a new section, called **NETWORK**. This value will be set
+during *engine-setup* - refer to [this section](#engine-setup) - and the
+default choice will be 'yes', which will result in all traffic being dropped to
+the VMs.
+
+Existent ports can later be updated, disabling the **port-security** attribute,
+which would remove the ACLs that drop all IP traffic to the VM.
+
+As defined in the [network port security attribute definition](https://developer.openstack.org/api-ref/network/v2/#port-security), updating the **port-security** attribute on a network object **does not** cascade the value to ports attached to that network - meaning that effect would only apply to ports created *after* that update. Check the [upgrade](#installation/upgrade) section for the consequences of this requirement on ovirt-provider-ovn upgrades.
+
+## User workflow
+
+Since the feature is active by default, the intended workflow - from the oVirt
+engine user's perspective - is:
+
+* create an external network on the ovirt-provider-ovn - its default
+*port_security_enabled* value is true - which can be overriden during engine-setup.
+* create VMs attached to this external network - they will inherit the
+*port_security_attribute* from the network, which will result in dropping all
+IP traffic to that VM.
+* the intended traffic will have to be white-listed. Assuming the user is
+interested in allowing ssh traffic, the following ansible playbook should be
```

Comment:
used where ever you want:; just point it at the ovirt-provider-ovn. 

That can be deployed any way the user wants - but in the ovirt assisted deploy, we assume it is deployed in the same host as the engine.

I don't think this belongs here, since the user could be using this without ovirt.